src/js/administration/hooks.ts

```diff
@@ -10,3 +14,11 @@ function fetchSettings() {
 export function useFetchSettings() {
     return useQuery("settings", fetchSettings);
 }
+
+export function useCheckAdminRole(requiredRole: AdministratorRoles) {
+    const { data: account, isLoading }: { data: Account; isLoading: boolean } = useGetAccount();
```

Comment:
Isn't typing inferred here or is return type for `useGetAccount` not defined?

Reply:
This was done since the default typing being returned for `useQuery` had the datatype as `unknown`. That said the reactQueryAPI provides for a way to specify the returned data type. Next push will update the queries typing so that types can be automatically inferred for all usages of the request